aten/src/ATen/native/sparse/cuda/SparseCUDABlas.cu

```diff
@@ -81,7 +76,7 @@ cusparseOperation_t convertTransToCusparseOperation(char trans) {
   }
 }
 
-#if !defined(_MSC_VER) && defined(__CUDACC__) && CUSPARSE_VERSION >= 10301 // CUDA release >= 10.2 and not windows
+#if defined(__CUDACC__) && CUSPARSE_VERSION >= 11000
```

Comment:
This would disable this feature on Linux with CUDA-10.2, isn't it?

Reply:
cuda 10.2 on linux will use the old API, which is still functionally the same